libtest/main.cc

```diff
@@ -332,7 +332,9 @@ int main(int argc, char *argv[], char* environ_[])
     do
     {
       exit_code= EXIT_SUCCESS;
-      fatal_assert(sigignore(SIGPIPE) == 0);
+      struct sigaction sa;
+      sa.sa_handler=  SIG_IGN;
```

Comment:
There should only be one space after the equals sign on this line, I think. Not that it really matters....